AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -138,6 +138,8 @@ greenweez.com#$##staticBackdrop { display: none !important; }
 greenweez.com#$#.modal-backdrop { display: none !important; }
 greenweez.com#$#body { overflow: visible !important; }
 foerde-sparkasse.de##.if6_eprivacy
+wetter.com#$##cmp-style-reset { display: none !important; }
```

Comment:
hiding breaks videos

Reply:
It breaks videos at least in Safari, we need to accept cookies

existing rule doesn't work for some reasons
`wetter.com#%#AG_onLoad(function(){if("undefined"!=typeof localStorage)try{if(!localStorage.getItem("uc_tcf")){var b=new MutationObserver(function(){var a=document.querySelector("button#cmp-btn-accept");a&&setTimeout(function(){a.click();b.disconnect()},1E3)});b.observe(document,{childList:!0,subtree:!0});setTimeout(function(){b.disconnect()},1E4)}}catch(a){}});`